README.md

```diff
@@ -375,6 +368,40 @@ const serviceCustomizations = {
 const service = new Pryv.Service(serviceInfoUrl, serviceCustomizations);
 ```
 
+##### Usage of Pryv.Service.
+
+See: [Pryv.Service](https://pryv.github.io/js-lib/docs/Pryv.Service.html) for more details
+
+- `service.info()` - returns the content of the serviceInfo in a Promise 
+
+  ```javascript
+  // example: get the name of the platform
+  const serviceName = await service.info().name
+  ```
+
+- `service.infoSync()` - returns the cached content of the serviceInfo, requires `service.info()` to be called first.
+
+- `service.apiEndpointFor(username, token)` Will return the corresponding API endpoint for the provided credentials, `token` can be omitted.
+
+##### Assets & Visual Usage 
+
+To customize assets and visual  refer to [pryv.me assets github](https://github.com/pryv/assets-pryv.me).
+
+To customize the Sign in Button refer to [Sign in Button in pryv.me assets](https://github.com/pryv/assets-pryv.me/tree/master/lib-js/#buttonsignin).
+
+#### Platfrom-specific ressources 
+
+can be used in a browser with `setAllDefaults()`
+
+This will load the `css` and `favicon` properties of assets definitions.
+
+```javascript
+(async function () {
+  const service = await Pryv.Browser.setupAuth(authSettings, serviceInfoUrl);
+  (await service.assets()).setAllDefaults(); // will load the default Favicon and CSS for this platform
+})();
+```
+
 ##### Pryv.Browser
```

Comment:
this part does not make sense anymore in its context.

Reply:
Ok .. reshaping